client/my-sites/checkout/checkout/index.jsx

```diff
@@ -287,6 +288,13 @@ export class Checkout extends React.Component {
 			cartItem = domainMapping( { domain: cartMeta } );
 		}
 
+		if ( startsWith( this.props.product, 'quickstart-session' ) ) {
```

Comment:
If we're making this so that the product gets added to the cart by visiting a URL, what do you think about using the existing, and more generic, "support-session" name for when it's shown to the user outside of sales copy, like in URLs?

"Support session" was meant to be a generic enough name that would allow us to not need to change it when running various tests around how we talk about the sessions. Like the current "quickstart" copy.

"concierge session" was more of an internal name, and how the product is referenced in the code.

In the future we might try to sell them as "SEO session", or "Marketing session", or whatever. But those could all be generally referred to as a "Support session".

That was the original intent anyway. Feel free to ignore if you feel strongly against it.

Reply:
This is a good point. I have implemented this in [8ba81a6](https://github.com/Automattic/wp-calypso/pull/33749/commits/8ba81a6719dad784e97f993d70fd7f61dd7b9c7c).

I have used `concierge-session` in the URL, as you suggested in [this comment](https://github.com/Automattic/wp-calypso/pull/33749#discussion_r296068147). If you feel it's better to use support-session, I can quickly change that too.